### DISCUSSION TITLE
Detecting User Typing Status

### DISCUSSION THREAD

**Discussion by JiXiangChua** [python-telegram-bot/python-telegram-bot#4443] [Q&A] [answered]
Hi, do the python-telegram-bot have methods or APIs we can use for detecting if the user is still typing a message?  

The motivation is in certain scenarios, we might want to capture the entire message before sending the full content to our server. Sometimes, when we chat with each other, we broke down our message into multiple chat bubbles. So, if the user is still typing, I do not want to send the messages received to my server yet.

Thanks!

**Comment by harshil21** [comment, marked as answer]
Hi,

there is no way to detect that unfortunately.

**Reply by JiXiangChua** [reply]
I see! Okay thanks!
